literature/pajkossy_2001_double_3063/pajkossy_2001_double_3063_p4_5A.yaml

```diff
@@ -0,0 +1,102 @@
+    comment: # Add whatever you think is important to understand the data
+
+electrochemical system:
+    version: 1
+    instrumentation:
+        - type:
+          supplier: 
+        - type:
+          model:
+          supplier:
+        - type:
+          model:
+          supplier:
+    electrolyte:
+        type: aq # aqueous = aq, non-aqueous=naq
+        components: # create as many components as necessary. A simple electrolyte consists of two components, i.e., H2SO4 and water
+            - name: KClO4 # can be trivia name, sum formula, etc
+              concentration:
+                  value: 0.1
+                  unit: M #[M, mM, µM, g kg-1, ...]
+              type: #
+              source:
+                  supplier:
+                  LOT: 
+            - name: water
+              type: solvent
+              proportion:
+                  value:
+                  unit:
+              source:
+                  refinement:
+                  quality:
+                  total organic carbon: 
+                      value:
+                      unit:
+            - name: 
+              type:
+              proportion:
+                  value: 
+                  unit:
+            - name: 
+              type: 
+              source:
+                  supplier:
+                  LOT: 
+              concentration:
+                  value:
+                  unit: #[M, mM, µM, g kg-1, ...] 
+        ph: 
+            value:
+            uncertainty:
+        temperature: # provide the temperature as it is given in the article. Assume room temperature if nothing is provided
+            value: 298.15 
+            unit: K # [K, C, F]
+    electrodes:
+        configuration: 3 # number of electrodes
+        reference electrode:
+            type: SCE # the reference electrode given in the experimental section. Might differ from that in the plot, MSE-sat, MSE-3M, MSE-1M, RHE, AgCl2
+            source:
+                supplier:
+        counter electrode:
+            material: Pt
+            crystallographic orientation:
+            shape: wire
+        working electrode:
+            material: Pt
+            crystallographic orientation: '111' # as STRING hkl, 100, 110, poly
+            source:
+                supplier:
+                LOT: 
+            shape: # optional
+                type: # bead/sphere
+                height:
+                    value:
+                    unit:
+                diameter:
+                    value:
+                    unit:
+            geometric electrolyte contact area: 
+                value:
+                unit:
+            preparation procedure: the electrode was annealed in a hydrogen or propane gas flame and, after cooling in nitrogen for about 1 min, it was immersed in water and thereafter transferred to the electrochemical cell with a protecting water droplet # Short description of sample preparation. This can be very useful. :)

```

Comment:
```suggestion
            preparation procedure: The electrode was annealed in a hydrogen or propane gas flame and, after cooling in nitrogen for about 1 min, it was immersed in water and thereafter transferred to the electrochemical cell with a protecting water droplet. # Short description of sample preparation. This can be very useful. :)

```